tests/run-make/rust-lld-by-default-beta-stable/rmake.rs

```diff
@@ -12,11 +12,7 @@ use run_make_support::rustc;
 fn main() {
```

Comment:
cc @rust-lang/release, i don't have a good way to test changes to this file. i tried setting `channel = "beta"` in config.toml and it tried to build llvm enzyme and a bunch of stuff failed.

Reply:
@jyn514 Are you by chance on macos or windows? I didn't add support for them yet. But also there's `#enzyme = false` as default in config.example.toml, which I assume you didn't change? ThenI probably messed something up, because it shouldn't build enzyme. Can you share your exact config?